release/_bom.sh

```diff
@@ -273,6 +309,19 @@ function generate_pom_release_bom_third_party {
 	) >> "${pom_file_name}"
 }
 
+function generate_pom_release_distro {
```

Comment:
@pgydev, since the `generate_distro_jar` is a demand exclusive of this function, why not invoke it here instead of invoke it on the `build_release`? Don't you think we can encapsulate this logic here?

Reply:
I see, @pgydev. :slightly_smiling_face: But, does it make sense we run the `generate_distro_jar` and not run the `generate_pom_release_distro`? What do you think? :slightly_smiling_face: 

We can chat more about it today before our daily, what do you think?